backend/server.js

```diff
@@ -196,22 +197,60 @@ app.get('/api/users', async (req, res) => {
 // Contact
 app.post('/api/contact', async (req, res) => {
   const { name, email, subject, message } = req.body
+
+  if (!name || !email || !subject || !message) {
+    return res.status(400).json({ message: 'All fields are required' });
+  }
```

Comment:
I don't think this is required? Because when we create the Contact Mongoose will automatically throw an error if there is a missing field. Could you try removing this and see if it still works, just so it's not redundant